### DISCUSSION TITLE
Accessing results with simple types

### DISCUSSION THREAD

**Discussion by normelton** [prisma/prisma#20421] [Q&A] [answered]
I've got a simple schema:

```prisma
model User {
  id            Int    @id @default(autoincrement()) @db.UnsignedInt
  supervisor_id Int?   @db.UnsignedInt
  email         String @db.VarChar(64)
  name          String @db.VarChar(64)
  team          String @db.VarChar(32)

  @@map("users")
}
``` 

I load a user...

```typescript
const user = prisma.user.findUnique({
  where: {
    id: 12
  }
})
``` 

Beautiful, elegant, clean. But if I want to pass that user to a function, it seems I need this:

```typescript
function select(user: Prisma.Prisma__UserClient<(GetResult<{ id: number; supervisor_id: number | null; email: string; name: string; team: string; }, unknown> & {}) | null, null, DefaultArgs>) {
  ...
}
```

This seems ... inelegant. Am I missing something here? Is there no way for a result to simply be `user: User`?

Apologies if I've missed something obvious.

**Comment by ludralph** [comment]
Hi @normelton 👋 

`GetResult` was an unintended change in Prisma version 4.16.1 and it would be changed in Prisma v5.1.0. You can see this comment [here](https://github.com/prisma/prisma/discussions/20009#discussioncomment-6516107).

You can also try the suggestion in the discussion linked above.

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Reply by normelton** [reply]
Got it, thanks. I'll close this discussion!

**Reply by nikolasburk** [reply, marked as answer]
Hey @normelton 👋 we fixed some issues related to the `GetResult` type in yesterday's release. You might want to upgrade to [`5.1.0`](https://github.com/prisma/prisma/releases/tag/5.1.0) to see if you're still seeing this behaviour 🙌

**Comment by normelton** [comment]
Moving this troubleshooting to #20009.
